packages/react/src/textarea/Textarea.css.ts

```diff
@@ -0,0 +1,80 @@
+import { theme } from "../styles";
+import { style } from "../vanilla-extract";
+import { recipe } from "../vanilla-extract";
+
+export const wrapper = recipe({
+  base: [
+    {
+      border: "1",
+      fontSize: "md",
+      overflow: "auto",
+      rounded: "sm",
+    },
+    style({
+      border: "1",
+      borderColor: theme.colors["border.default"],
+      gap: theme.spacing["xs"],
+      letterSpacing: "-.14px",
```

Comment:
Do we need this?

Reply:
I coded it based on the figman file
![alt text](image.png)

Reply:
Not sure if we need it or not. 
C.C @radnan vai.

Reply:
we shouldn't mess with letter spacing except for maybe headings

Reply:
@JafarSaifullah